topics/items.md

```diff
@@ -17,41 +17,49 @@ body_class: items
+| `has_container`                   | `VARCHAR(255)`  | The parent container GUID of the item - could be the list itself, or a section within the list. Prefixed with the type `tenantLists` or `tenantSections` to mark out type                                       | `tenantSections:9CBA2921-17BD-0A62-7942-421DCD8E79E6` |
+| `date_added`                      | `TIMESTAMP`     | Date the item was added to the list                                                                                                                                                                             | `2016-01-12 00:00:00.0`                               |
+| `library_note`                    | `VARCHAR(4096)` | Textual information about the item only visible to staff                                                                                                                                                        | `Example library note`                                |
+| `student_note`                    | `VARCHAR(4096)` | Textual information about the item visible to all users                                                                                                                                                         | `Example student note`                                |
+| `year_added`                      | `INTEGER`       | Year of when the list was created                                                                                                                                                                               | `2018`                                                |
+| `month_added`                     | `INTEGER`       | Month of when the list was created, values `1-12`                                                                                                                                                               | `12`                                                  |
+| `week_added`                      | `INTEGER`       | ISO 8601 week of when the list was created, note Monday is the start of the ISO 8601 week, values `1-53`                                                                                                        | `52`                                                  |
+| `dow_added`                       | `INTEGER`       | Day of week of when the list was created, values `0-6`, Sunday is `0`                                                                                                                                           | `5`                                                   |
+| `primary_web_address`             | `VARCHAR(4096)` | If a chapter or similar 'part' is present in a bookmark, this will be the link of the chapter                                                                                                                   |                                                       |
```

Comment:
[comment] I took these descriptions more or less as written in the all list items [support article](https://support.talis.com/hc/en-gb/articles/17338837081885-All-List-Items-Report). Let me know if they need tweaking.